Octokit.Tests.Integration/Clients/PullRequestsReviewRequestClientTests.cs

```diff
@@ -9,30 +9,40 @@
 
 public class PullRequestsReviewRequestClientTests
 {
-    const string collaboratorLogin = "octocat";
+    private static readonly string[] _collaboratorLogins = {
+        "octocat",
+        "gdziadkiewicz-test",
+        "ryangribble-testorg"
```

Comment:
this is my test organization rather than a user, which is causing tests to fail.

Also it should be possible to prove pagination works with only 2 collaborators (just adjust the pagesizes in the ApiOptions in the tests to 1 rather than 2)